### ISSUE TITLE
Group points depending on zoom level

### ISSUE THREAD

**Issue by arski** [shutterstock/rickshaw#83]
Hi there,

Maybe this is possible, but I don't think so.. What I would like to happen is for a bunch of data points, submitted with corresponding timestamps, to be grouped by week/month/year depending on the zoom level.. So for example if I have hundreds of data points per month for about a year, I would like the default view in a bar plot e.g. to have 12 bars, one for each month, with the total value of all the points inside that month.. When I zoom in, it might then show data by week or day.. then by hour/minute.. 

Basically, this entails allowing to split series data into points that have their own title and the series name and then grouping these together.

I'll try and look into it, but any help would be appreciated.

Cheers,
Martin

**Comment by RichardLitt** [comment]
This is a duplicate of #41. I would say go for it, if you're interested in helping out!
